### PR TITLE
Updating Workspace styling to match new designs

### LINKED ISSUE #160: [terra-application] Workspace theming revision request
# Feature Request
Revisions to the design of tabs within workspace have been requested.

## Description
UX has presented news design revisions to theme of the tab components within workspace. In accordance with this we will need to adjust the implementation of the tab itself. This may result in new or deleted theme-able variables.

## @ Mentions
@neilpfeiffer @ryanthemanuel @mjhenkes @tbiethman

### PR DESCRIPTION
**Pull request by tbiethman**
### Summary

Updating Workspace styling to match new designs

Closes #160.

### Deployment Link

https://terra-applic-visual-rev-u8uoee.herokuapp.com/tests/terra-application/workspace/workspace

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: src/workspace/clinical-lowlight-theme/TabDropDown.module.scss
Comment (mjhenkes) [inline src/workspace/clinical-lowlight-theme/TabDropDown.module.scss]: no px and double ';'? seems like a typo.

File: src/workspace/subcomponents/Tab.module.scss
Comment (mjhenkes) [inline src/workspace/subcomponents/Tab.module.scss]: TODO still needed?

Comment (dkasper-was-taken) [reply src/workspace/subcomponents/Tab.module.scss]: Updated.

File: src/workspace/clinical-lowlight-theme/TabDropDown.module.scss
Comment (dkasper-was-taken) [reply src/workspace/clinical-lowlight-theme/TabDropDown.module.scss]: Updated.

**Review by mjhenkes — APPROVED** [review]

File: src/workspace/orion-fusion-theme/Tab.module.scss
Comment (neilpfeiffer) [inline src/workspace/orion-fusion-theme/Tab.module.scss]:
```suggestion
    --terra-application-workspace-tab-border: 1px solid #868a8c;
```

Comment (dkasper-was-taken) [reply src/workspace/orion-fusion-theme/Tab.module.scss]: Updated

**Review by neilpfeiffer — APPROVED** [review]
Tested: PR #169
Resolves: Issue #160

**Reviewed:**
- theme changes reviewed by @sethbroox https://github.com/cerner/terra-application/pull/169#issuecomment-810508819 and @neilpfeiffer 


Verified, ready to merge after additional or technical review or maintainer review.

**UX Reviewed** |
-----------|

File: src/workspace/subcomponents/_Tabs.jsx
Comment (yuderekyu) [inline src/workspace/subcomponents/_Tabs.jsx:191]: thoughts on saving the workspace node for reuse/readability?
```js
const workspaceNode = this.containerRef.current.parentNode.parentNode;
const workspaceStyle = window.getComputedStyle(workspaceNode, null);
...
const workspaceRect = workspaceNode.getBoundingClientRect();
```

**Review by yuderekyu — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

**Pull request merged**
